HadesScheduler/k8s/namespace.go

```diff
@@ -0,0 +1,46 @@
+package k8s
+
+import (
+	"context"
+
+	"log/slog"
+
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func createNamespace(ctx context.Context, clientset *kubernetes.Clientset, namespace string) (*corev1.Namespace, error) {
+	// Create a namespace in the Kubernetes cluster
+	slog.Info("Creating namespace", "namespace", namespace)
```

Comment:
I feel like we're spaming the console too much if we always have a `creating ...` and `created ...` log message at the info channel. Personally, I prefer the `created ...` log msg more and would put only this in the Info output
```suggestion
	slog.Debug("Creating namespace", "namespace", namespace)
```

Reply:
I agree